examples/pubsub/README_pubsub_tpm_keystore.txt

```diff
@@ -0,0 +1,103 @@
+1. TESTED ENVIRONMENT:
+Apollo Lake processor-1.60GHz
+OS - Ubuntu - 20.04.2 LTS
+Kernel - 5.4.0-74-generic
+
+2. ENVIRONMENT SETUP:
+Follow the below steps in both Publisher and Subscriber nodes.
+Install required packages:
+    sudo apt -y install acl autoconf autoconf-archive automake build-essential cmake doxygen gcc git iproute2 libcurl4-openssl-dev libjson-c-dev libcmocka0 libcmocka-dev libglib2.0-dev libini-config-dev libmbedtls-dev libssl-dev libsqlite3-dev libtool libyaml-dev pkg-config procps python3-pip sqlite3 udev uthash-dev
+
+Install the TPM2 Software Stack library:
+    cd ${HOME}
+    git clone https://github.com/tpm2-software/tpm2-tss.git
+    cd ${HOME}/tpm2-tss
+    ./bootstrap
+    ./configure --with-udevrulesdir=/etc/udev/rules.d --with-udevrulesprefix=70-
+    make -j$(nproc)
+    sudo make install
+    sudo ldconfig
+    sudo udevadm control --reload-rules && sudo udevadm trigger
+
+Install the TPM2 Tools (tpm2-tools):
```

Comment:
Please do not use the prepackaged version of tpm2-tools (ubuntu 20.04 has an incompatible version of tpm2-tools for tpm2-pkcs11). Use release tags where possible for reproducing the same environment setup.